internal/indexer/definitions/filelist.yaml

```diff
@@ -67,6 +67,6 @@ irc:
 
     match:
       infourl: "/details.php?id={{ .torrentId }}"
-      torrenturl: "/download.php?id={{ .torrentId }}&file={{ .torrentName }}.torrent&passkey={{ .passkey }}"
+      torrenturl: "/download.php?id={{ .torrentId }}&passkey={{ .passkey }}"
       encode:
         - torrentName
```

Comment:
Can also remove this since URL safe encoding for `torrentName` is not needed anymore due to removal.